# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1312447619 -7200
#      Thu Aug 04 10:46:59 2011 +0200
# Node ID fe30859fb540f40636dc82c1942eb0432cf106da
# Parent  83e83a70438330ea7404584c5c63bfa58c97b3da
IRC send messages are being displayed.

diff --git a/gui/src/qirc/IRCClientImpl.cpp b/gui/src/qirc/IRCClientImpl.cpp
--- a/gui/src/qirc/IRCClientImpl.cpp
+++ b/gui/src/qirc/IRCClientImpl.cpp
@@ -424,17 +424,17 @@ IRCClientImpl::handleIncomingLine (const
             }
         }
     }
 }
 
 void
 IRCClientImpl::sendLine (const QString &line)
 {
-  emit debugMessage (QString ("<<<send: \"%1\"").arg (line));
+  emit debugMessage (QString (">>>send: \"%1\"").arg (line));
   if (m_connected)
     m_tcpSocket.write ((line + "\r\n").toStdString ().c_str ());
 }
 
 void
 IRCClientImpl::sendIRCCommand (const QString &command, const QStringList &arguments)
 {
   QString line = command;
